scipy/spatial/_spherical_voronoi.py

```diff
@@ -15,43 +15,43 @@
 import scipy
 import itertools
 from . import _voronoi
-from scipy.spatial.distance import pdist
+from scipy.spatial import cKDTree
 
 __all__ = ['SphericalVoronoi']
 
+
 def sphere_check(points, radius, center):
     """ Determines distance of generators from theoretical sphere
     surface.
 
     """
-    actual_squared_radii = (((points[...,0] - center[0]) ** 2) +
-                            ((points[...,1] - center[1]) ** 2) +
-                            ((points[...,2] - center[2]) ** 2))
+    actual_squared_radii = (((points[..., 0] - center[0]) ** 2) +
+                            ((points[..., 1] - center[1]) ** 2) +
+                            ((points[..., 2] - center[2]) ** 2))
     max_discrepancy = (np.sqrt(actual_squared_radii) - radius).max()
     return abs(max_discrepancy)
 
-def calc_circumcenters(tetrahedrons):
-    """ Calculates the cirumcenters of the circumspheres of tetrahedrons.
 
+def calc_circumcenters(tets):
+    """ Calculates the cirumcenters of the circumspheres of tetrahedra.
     An implementation based on
     http://mathworld.wolfram.com/Circumsphere.html
-
+    This function is not used in the SphericalVoronoi class but is kept for
+    backwards compatibility.
```

Comment:
The module itself is private, so I suspect we may be able to simple remove if we really don't need it -- @rgommers can likely confirm